apps/sr-frontend/src/app/App.tsx

```diff
@@ -28,6 +29,7 @@ export default function App() {
               <Stack gap={1} pt={1}>
                 <CharacterSelector />
                 <Character />
+                <RelicInventory />
```

Comment:
Wrap this in an Accordion (see stuff in Character)

Reply:
I don't want to wrap the inner contents, so we can more easily transplant it into a separate page later down the line

Reply:
fixed on ba2186105d0f9a6fc46112936bd0b5dda574e2c7